### ISSUE TITLE
Is this project abandoned? 

### ISSUE THREAD

**Issue by ZueffC** [jmoiron/sqlx#908]
Hi folks! As I see there isn't any activity in repository for the whole four years. 
Is this project closed or moved to another git-like CVS or it's just abandoned?

**Comment by uvulpos** [comment]
The author mentioned once that he is done with the project and won't do anything else. That's why I created a fork you might be interested in. It's not actively maintained by me, but I do at least update dependencies and check and merge pull requests, so if you find a bug and have a fix for it, share it with the community :)

https://github.com/go-sqlx/sqlx

Hope that helps you :)

**Comment by uvulpos** [comment]
@dtrzepala @TheBotlyNoob @gmdias727 @yudintsevegor @Relvox @lcs86-dev @roger-russel @hirasawayuki @BruAPAHE @guitarpawat @fschuermeyer might also be interested in that 🙂

**Comment by uvulpos** [comment]
in #878 I also mentioned the post: https://www.reddit.com/r/golang/comments/l43wn7/comment/gkm42xz/?utm_source=share&utm_medium=web2x&context=3

**Comment by Aklscc** [comment]
> The author mentioned once that he is done with the project and won't do anything else. That's why I created a fork you might be interested in. It's not actively maintained by me, but I do at least update dependencies and check and merge pull requests, so if you find a bug and have a fix for it, share it with the community :)
> 
> https://github.com/go-sqlx/sqlx
> 
> Hope that helps you :)

good job!

**Comment by dlsniper** [comment]
Hello! Sorry for the delay in replying. @ardan-bkennedy and I recently stepped in to help maintain the project.
We are in the process of triaging all the opened PRs and then issues, and we plan to address outstanding issues and upgrade the library to support generics and new Go versions, among other things.

**Comment by uvulpos** [comment]
@dlsniper sounds good :) 
But what I would like to suggest, is still moving the upcoming development to our fork go-sqlx because it's more convenience for people to see a project in a GitHub organization instead of a user (no offense). 

If you are interested in that, I would invite you to join the organization and even transfer it after some time to you, when I'm convinced updates are continuously merged and it's not just a phase 🙂

**Comment by uvulpos** [comment]
Ok, works for me 🙂

**Comment by smlx** [comment]
@dlsniper is this work taking place on the `update-dependencies` branch?
